docs/startup/preview/index.md

```diff
@@ -527,4 +527,5 @@ Take note of the following issues with the embedded deployment feature for the 0
     * Number of Read/Writes
     * Leader reelections
     * more..
-* The LocalApplicationRunner does not yet support the low level API. This means you cannot use StreamTask with LocalApplicationRunner.
\ No newline at end of file
+* The LocalApplicationRunner does not yet support the low level API. This means you cannot use StreamTask with LocalApplicationRunner.
+* Currently you can only run a single applicaiton with a single zookeeper.
```

Comment:
I thought this limitation was only for LinkedIn. Can't we workaround this issue by using a unique `job.id` or `app.id`??